### DISCUSSION TITLE
getting the element where the handler was triggered?

### DISCUSSION THREAD

**Discussion by hanoii** [alpinejs/alpine#888] [1. Help]
I think I am trying to do something simple although not sure what I am missing or how to do it.

If you have

```
<a x-on:click="myFunction($event)" href="#"><span>Link</span></a>
```

`myFunction` gets called properly, but $event.target has the span element.

I can't find a standard way of acting on the element that actually has the onclick handler. I want to do something with the a element, not the span element.

`click.self` doesn't trigger the function as the click stills bubble up from the span. I would have expected this to work exactly as jquery where $('a').click() gets the a element.

Any help is appreciated!

**Comment by KevinBatdorf** [comment]
You could tell the browser to ignore the pointer events.

```html
<span style="pointer-events:none">Link</span>
```
